### ISSUE TITLE
TensorRT implementation

### ISSUE THREAD

**Issue by Darth-Kronos** [XuJiacong/PIDNet#54]
Find the link for the TensorRT implementation of PIDNet-S and PIDNet-M where the inference speed increases by 30% below:
[Link](https://github.com/Darth-Kronos/PIDNet_TensorRT)

**Comment by thom966** [comment]
Thank you so much for your sharing.
Is it possible to inference the TensorRT model to measure mIoU?

**Comment by Darth-Kronos** [comment]
Yes, it can be done. We'll be adding these to our repo soon. The link to the new repo: [link](https://github.com/Darth-Kronos/PIDNet_TensorRT)
